packages/desktop-client/src/components/rules/ActionExpression.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import { type CSSProperties } from 'glamor';
```

Comment:
Is it possible to use React's CSSProperties here instead? That would make any future work to migrate away from glamor easier.

Reply:
Makes sense: https://github.com/actualbudget/actual/pull/1517/commits/b55a117447bbcdba0daf765e3993b0135daab872